presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/thrift/HiveMetastore.java

```diff
@@ -63,11 +64,19 @@
 
     Optional<Database> getDatabase(MetastoreContext metastoreContext, String databaseName);
 
-    void addPartitions(MetastoreContext metastoreContext, String databaseName, String tableName, List<PartitionWithStatistics> partitions);
+    MetastoreOperationStats addPartitions(
```

Comment:
It is not that long. Keep it one line to reduce the diff. Same for the changes below.

Reply:
:) I like short ones. Presto thinks < 200 characters is OK, which is definitely too long. I try to make it lower than 120. Should we find a consistent number?